lib/audit_help.c

```diff
@@ -44,6 +45,23 @@ void audit_help_open (void)
 	}
 }
 
+/*
+ * This takes a string and replaces the old character with the new.
+ */
+static inline char *strtr (char *str, char old, char new)
+{
+	if (str == NULL) {
+		return NULL;
```

Comment:
I prefer if this API doesn't check for NULL.  Just like all string APIs, this should require non-null input.

Reply:
This is to keep static analysis tools quiet. Looking around, I don't see __attribute__((nonnull)) in the lib/attr.h header. Are you suggesting to just assume it can't be null with no compiler reinforcement?